grails-app/controllers/au/org/ala/biocollect/NoCacheFilterInterceptor.groovy

```diff
@@ -1,20 +1,23 @@
 package au.org.ala.biocollect
 
+import grails.interceptors.Matcher
 
 class NoCacheFilterInterceptor {
 
     private static final String HEADER_PRAGMA = "Pragma";
     private static final String HEADER_EXPIRES = "Expires";
     private static final String HEADER_CACHE_CONTROL = "Cache-Control";
-
+    private Matcher matcher
     int order = 1
     NoCacheFilterInterceptor() {
-        matchAll().excludes(controller:'document', action:'download')
+        matcher = matchAll().excludes(controller:'document', action:'download')
+            .excludes(uri: '/proxy/speciesLists/.*').excludes(uri: '/proxy/speciesList/.*')
+            .excludes(uri: '/proxy/speciesItemsForList/.*').excludes(uri: '/proxy/speciesProfile/.*')
```

Comment:
This looks fine but I had an issue with a matchAll with multiple excludes - see the ApiKeyInterceptor in ecodata.

Reply:
A quick test in staging shows the no-cache headers being applied to the species list
<img width="665" height="338" alt="image" src="https://github.com/user-attachments/assets/2d055bd3-e800-4885-8ca1-11332c70e1b9" />